### PR TITLE
data: SurajPratap10

### PR DESCRIPTION
**Pull request by SurajPratap10**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5030"></a>

### CODE REVIEW

File: data/SurajPratap10.json
Comment (imbeshat) [inline data/SurajPratap10.json:6]:
```suggestion
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
We don't need these now. Please, update these

File: data/SurajPratap10.json
Comment (SurajPratap10) [reply data/SurajPratap10.json]: removed both of the unwanted sections

**Review by imbeshat — APPROVED** [review]

**Comment by SurajPratap10** [comment]
won't it get MERGED?

**Comment by imbeshat** [comment]
> By mistake I closed it, I wanted to ask that won't it get MERGED.

It will be merged only after it gets approval from maintainers

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Comment by SurajPratap10** [comment]
> LGTM 🎉

Thanks

**Comment by SurajPratap10** [comment]
added "tags" section in the latest commit

**Review by priyankarpal — APPROVED** [review]
😎

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**
